### ISSUE TITLE
03. Implement 4 loss functions

### ISSUE THREAD

**Issue by DevKiHyun** [DevKiHyun/SRNTT.pytorch#5]
Open issue
